src/engine/strat_engine/backstore/util.rs

```diff
@@ -33,9 +34,12 @@ pub fn get_udev_block_device(
     let mut enumerator = libudev::Enumerator::new(&context)?;
     enumerator.match_subsystem("block")?;
 
+    // Get canonical form to ensure we do correct lookup in udev db
+    let canonical = fs::canonicalize(dev_node_search)?;
```

Comment:
Why is it necessary to canonicalize the path? Does this have something to do with multipath? If not, it should be taken out of this commit.